INFO - 2024-12-09 01:40:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-12-09 01:51:47 --> Error connecting to the database: mysqli_sql_exception: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('192.168.1.104', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:51:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Host '192.168.1.100' is not allowed to connect to this MariaDB server
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
ERROR - 2024-12-09 01:56:08 --> Error connecting to the database: mysqli_sql_exception: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('192.168.1.104', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:56:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Host '192.168.1.100' is not allowed to connect to this MariaDB server
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
ERROR - 2024-12-09 01:56:11 --> Error connecting to the database: mysqli_sql_exception: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('192.168.1.104', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:56:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Host '192.168.1.100' is not allowed to connect to this MariaDB server
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 01:56:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 01:56:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-12-09 01:56:20 --> Error connecting to the database: mysqli_sql_exception: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('192.168.1.104', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:56:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Host '192.168.1.100' is not allowed to connect to this MariaDB server
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
ERROR - 2024-12-09 01:58:34 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 187)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('http://192.168....', 'root', 'astagina', 'catat', 3306, '', 0)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#7 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('http://192.168....', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:58:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
ERROR - 2024-12-09 01:58:43 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 187)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('http://192.168....', 'root', 'astagina', 'catat', 3306, '', 0)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#7 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('http://192.168....', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:58:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
ERROR - 2024-12-09 01:58:55 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 187)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('http://192.168....', 'root', 'astagina', 'catat', 3306, '', 0)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#7 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('http://192.168....', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:58:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
ERROR - 2024-12-09 01:59:14 --> Error connecting to the database: mysqli_sql_exception: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('192.168.1.104', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Host '192.168.1.100' is not allowed to connect to this MariaDB server in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:59:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Host '192.168.1.100' is not allowed to connect to this MariaDB server
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
ERROR - 2024-12-09 01:59:41 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\laragon\\www\\...', 187)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('https://astagin...', 'root', 'astagina', 'catat', 3306, '', 0)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#7 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:187
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(187): mysqli->real_connect('https://astagin...', 'root', 'astagina', 'catat', 3306, '', 0)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\catat\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\laragon\www\catat\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Auth->action_login()
#7 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\laragon\www\catat\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\catat\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\laragon\www\catat\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\laragon\www\catat\vendor\codeigniter4\framework\system\rewrite.php(44): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-12-09 01:59:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for https failed: No such host is known. 
[Method: POST, Route: action_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->action_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 02:06:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:12:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:12:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:17:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:17:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:24:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:24:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:27:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:27:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-09 02:27:20 --> ErrorException: Undefined variable $userData
[Method: GET, Route: user/pengumuman]
in APPPATH\Views\user\pengumuman.php on line 7.
 1 APPPATH\Views\user\pengumuman.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $userData', 'C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php', 7)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('user/pengumuman', [], true)
 5 APPPATH\Controllers\User.php(166): view('user/pengumuman')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->pengumuman()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 02:27:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:30:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:50:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:57:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:57:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:57:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:58:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:58:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:59:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 02:59:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 03:45:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 03:45:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 03:45:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 03:45:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 03:45:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 03:45:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-12-09 03:45:42 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 13.
 1 APPPATH\Views\admin\dashboard.php(13): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(84): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
WARNING - 2024-12-09 03:45:42 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 23.
 1 APPPATH\Views\admin\dashboard.php(23): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(84): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
WARNING - 2024-12-09 03:45:42 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 33.
 1 APPPATH\Views\admin\dashboard.php(33): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(84): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
WARNING - 2024-12-09 03:45:42 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 43.
 1 APPPATH\Views\admin\dashboard.php(43): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(84): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 03:59:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-12-09 03:59:24 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 13.
 1 APPPATH\Views\admin\dashboard.php(13): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(87): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
WARNING - 2024-12-09 03:59:24 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 23.
 1 APPPATH\Views\admin\dashboard.php(23): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(87): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
WARNING - 2024-12-09 03:59:24 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 33.
 1 APPPATH\Views\admin\dashboard.php(33): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(87): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
WARNING - 2024-12-09 03:59:24 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Views\admin\dashboard.php on line 43.
 1 APPPATH\Views\admin\dashboard.php(43): number_format(null, 0, ',', '.')
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\dashboard.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/dashboard', [], true)
 5 APPPATH\Controllers\Admin.php(87): view('admin/dashboard', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->dashboard()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 03:59:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 03:59:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-09 03:59:45 --> ErrorException: Undefined variable $no
[Method: GET, Route: admin/read_pengumuman]
in APPPATH\Views\admin\read_pengumuman.php on line 23.
 1 APPPATH\Views\admin\read_pengumuman.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $no', 'C:\\laragon\\www\\catat\\app\\Views\\admin\\read_pengumuman.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\read_pengumuman.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/read_pengumuman', [], true)
 5 APPPATH\Controllers\Admin.php(443): view('admin/read_pengumuman', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->read_pengumuman()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 04:00:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-09 04:00:03 --> ErrorException: Undefined variable $siswa
[Method: GET, Route: admin/read_pengumuman]
in APPPATH\Views\admin\read_pengumuman.php on line 26.
 1 APPPATH\Views\admin\read_pengumuman.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $siswa', 'C:\\laragon\\www\\catat\\app\\Views\\admin\\read_pengumuman.php', 26)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\read_pengumuman.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/read_pengumuman', [], true)
 5 APPPATH\Controllers\Admin.php(443): view('admin/read_pengumuman', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->read_pengumuman()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 04:00:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:02:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:02:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:03:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:04:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:04:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:06:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-09 04:06:54 --> ErrorException: Undefined variable $siswa
[Method: GET, Route: admin/read_pengumuman]
in APPPATH\Views\admin\read_pengumuman.php on line 50.
 1 APPPATH\Views\admin\read_pengumuman.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $siswa', 'C:\\laragon\\www\\catat\\app\\Views\\admin\\read_pengumuman.php', 50)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\admin\\read_pengumuman.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('admin/read_pengumuman', [], true)
 5 APPPATH\Controllers\Admin.php(443): view('admin/read_pengumuman', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Admin->read_pengumuman()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 04:07:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:28:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:29:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:29:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:30:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:30:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:30:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:31:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:32:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:32:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:32:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:33:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:33:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:33:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:35:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-09 04:35:43 --> ErrorException: Undefined array key "nama_pengumuman"
[Method: GET, Route: user/pengumuman]
in APPPATH\Views\user\pengumuman.php on line 23.
 1 APPPATH\Views\user\pengumuman.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nama_pengumuman"', 'C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php', 23)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('user/pengumuman', [], true)
 5 APPPATH\Controllers\User.php(175): view('user/pengumuman', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->pengumuman()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 04:36:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-09 04:36:39 --> ErrorException: Undefined array key "nama_pengumuman"
[Method: GET, Route: user/pengumuman]
in APPPATH\Views\user\pengumuman.php on line 35.
 1 APPPATH\Views\user\pengumuman.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nama_pengumuman"', 'C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php', 35)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('user/pengumuman', [], true)
 5 APPPATH\Controllers\User.php(175): view('user/pengumuman', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->pengumuman()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 04:37:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-09 04:37:30 --> ErrorException: Undefined array key "nama_pengumuman"
[Method: GET, Route: user/pengumuman]
in APPPATH\Views\user\pengumuman.php on line 35.
 1 APPPATH\Views\user\pengumuman.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nama_pengumuman"', 'C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php', 35)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\catat\\app\\Views\\user\\pengumuman.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('user/pengumuman', [], true)
 5 APPPATH\Controllers\User.php(175): view('user/pengumuman', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->pengumuman()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\laragon\\www\\catat\\public\\index.php')
INFO - 2024-12-09 04:37:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:39:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:41:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:44:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:47:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:48:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:48:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:49:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:50:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:50:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-09 04:50:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
